deploy/examples/create-external-cluster-resources.py

```diff
@@ -775,6 +806,9 @@ def gen_shell_out(self):
         return shOut
 
     def gen_json_out(self):
+        if self._arg_parser.dry_run:
+            return self._gen_output_map()
```

Comment:
I was wrong this line is printing `None` as here I'm returning empty json.
 How about if we print something like this in the end instead of `None` as sample json blob?
```[{"name": "rook-ceph-mon-endpoints", "kind": "ConfigMap", "data": {"data": null, "maxMonId": "0", "mapping": "{}"}}, {"name": "rook-ceph-mon", "kind": "Secret", "data": {"admin-secret": "admin-secret", "fsid": null, "mon-secret": "mon-secret"}}, {"name": "rook-ceph-operator-creds", "kind": "Secret", "data": {"userID": "client.healthchecker", "userKey": null}}, {"name": "rook-csi-rbd-node", "kind": "Secret", "data": {"userID": "csi-rbd-node", "userKey": null}}, {"name": "ceph-rbd", "kind": "StorageClass", "data": {"pool": "replicapool"}}, {"name": "monitoring-endpoint", "kind": "CephCluster", "data": {"MonitoringEndpoint": "", "MonitoringPort": null}}]```

Reply:
How about not printing anything at all?

Reply:
yeah sounds good.